fedbiomed/researcher/job.py

```diff
@@ -363,33 +366,48 @@ def start_nodes_training_round(self,
                 aggregator_args_thr_msg .
             secagg_arguments: Secure aggregation ServerKey context id
             do_training: if False, skip training in this round (do only validation). Defaults to True.
+            optim_aux_var: Auxiliary variables of the researcher-side Optimizer, if any.
         """
```

Comment:
I would add that `optim_aux_var` is for `declearn` optimizers only (not to be confused with `aggregator_args_thr_msg`  and `aggregator_args_thr_files`  that convey auxiliary variables too)